renv.lock

```diff
@@ -3057,7 +3057,7 @@
     },
     "renv": {
       "Package": "renv",
-      "Version": "1.0.9",
+      "Version": "1.1.3",
```

Comment:
@ayogasekaram this is the new version of {renv} I have on my machine. lets not go around with different versions! lets keep the most updated (may be more than the one I have)

Reply:
It looks like there's a "1.1.4" available on CRAN, I've just updated the files with that. Thank you for taking a look!